api/deployments/models/component_deployment.go

```diff
@@ -332,6 +337,46 @@ type ScheduledJobSummary struct {
 	// required: false
 	// example: "batch-abc"
 	BatchName string `json:"batchName,omitempty"`
+
+	// TimeLimitSeconds How long the job supposed to run at maximum
+	//
+	// required: false
+	// example: 3600
+	TimeLimitSeconds *int64 `json:"timeLimitSeconds,omitempty"`
+
+	// BackoffLimit Amount of retries due to a logical error in configuration etc.
+	//
+	// required: true
+	// example: 1
+	BackoffLimit int32 `json:"backoffLimit"`
+
+	// Resources Resource requirements for the job
+	//
+	// required: false
+	Resources ResourceRequirements `json:"resources,omitempty"`
+
+	//Node Defines node attributes, where pod should be scheduled
```

Comment:
format comment the same way the other objects have been

Reply:
Fixed